Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Link/Link.cpp

```diff
@@ -135,6 +135,18 @@ namespace AzToolsFramework
             return areOverridesPresent;
         }
 
+        PrefabDomConstReference Link::GetOverridePatch(AZ::Dom::Path path) const
```

Comment:
nit: GetOverridePatchAtExactPath might be a clearer name that indicates that the patch returned is from the exact path and not any child paths below